packages/sanity/src/presentation/useDocumentsOnPage.ts

```diff
@@ -15,16 +15,18 @@ type KeyedDocumentCache = Record<string, DocumentCache>
 
 const warnOnceAboutCrossDatasetReference = defineWarnOnce()
 
+/**
+ * @TODO should be refactored to an lru-cache that is keyed by the perspective, which could be an array (if it is, it should use consistent sorting),
```

Comment:
Do we need to create any stories for this / have they been created? :)

Reply:
We would need to create tickets, it's not proven yet it'll be needed tbh. I understand that the document preview stack in corel has improved, and it may not be necessary to run caches keyed to the perspective anymore 🤔

Reply:
Alright, I'm just concerned that it might fall through the cracks if there isn't a linear story to double check on this :)